docs/acceptance-criteria/uacs-rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,438 @@
+:toc: macro
+
+= Random Beacon v2 User Acceptance Criteria
+
+:icons: font
+:numbered:
+toc::[]
+
+== Purpose of the document
+This document stores User Acceptance Criteria for Random Beacon v2 feature. It
+will be used to gain common understanding of how Random Beacon v2 should act and
+will be utilised during the acceptance testing of the feature.
+
+== Usear Acceptance Criteria list
+
+[%header,cols="1,4"]
+|===
+| Area
+| UAC
+
+| Relay request
+| Anyone can send a relay entry request transaction.
+
+| Relay request
+| Processing of each relay entry request should always result in the generation
+  of a random number (relay entry) and sometimes also (if certain conditions are
+  met) in a creation of a new signing group.
+
+| Relay request
+| When sending the relay request, the requester should provide a fixed fee in T.
+
+| Relay request
+| Upon receiving the relay entry request, the random beacon contract should
+  determine validity of the request.
+
+| Relay request
+| When determining validity of the relay entry request, the following aspects
+  should be taken into account: +
+  - isn’t the other relay entry request already pending? +
+  - has the requester provided the exact required fee? (if the fee is lower or
+  higher, the transaction should be reverted) +
+  - is there at least one active signing group?
+
+| Relay request
```

Comment:
I updated the UACs based on what was agreed upon: e9cdc51244cf831adde0b177a9d2d3c7b21c35b2.

Reply:
Ok, so that I think is mostly in line whit what I added in that e9cdc51244cf831adde0b177a9d2d3c7b21c35b2 commit, but I missed a couple of places where I should have do the update. Fixed in c090c77d63a13e4caa536ea50f07f71ec435b5d6.